src/main/java/org/rumbledb/items/ItemFactory.java

```diff
@@ -62,6 +62,10 @@ public Item createDayTimeDurationItem(Period p) {
         return new DayTimeDurationItem(p);
     }
 
+    public Item createCurrentDateTimeItem() {
```

Comment:
I would avoid overloading the item factory with more specific functions, and rather use 

createDateTimeItem(new DateTime(), true)

on the caller side instead.

Otherwise, it creates a precedent and this class may become too big.

The same for the other two methods.

Reply:
Thank you for your comments.
I have made the changes, but this required to change the visibility modifiers for the constructors of DateTimeItem, DateItem and TimeItem. I checked with other item classes and the public modifiers seem to be used in the majority of places. Please let me know if this was not a desired change in this case.